scripts/nolints.json

```diff
@@ -72,7 +72,6 @@
  ["docBlame", "tactic_"],
  ["docBlame", "«termListΠ»"],
  ["docBlame", "«termListΣ»"],
- ["docBlame", "«term_⁻¹»"],
```

Comment:
Adding the new local notation meant that I would have needed to renamed this to `«term_⁻¹_1»` to get it to still silence the linter. I opted instead to just write a doc comment at the definition of the syntax.